OrbitClientModel/CaptureDeserializer.cpp

```diff
@@ -134,45 +90,47 @@ bool ReadMessage(google::protobuf::Message* message,
   return true;
 }
 
-CaptureData GenerateCaptureData(const CaptureInfo& capture_info, StringManager* string_manager) {
+void DoLoadCaptureInfo(const CaptureInfo& capture_info, CaptureListener* capture_listener,
```

Comment:
nit. Can "LoadCaptureInfo" be used instead of "DoLoadCaptureInfo"? Or is the "Do" meaningful here?

Reply:
done.